libcloud/compute/drivers/ec2.py

```diff
@@ -3129,6 +3139,12 @@ def create_volume(self, size, name, location=None, snapshot=None,
         if ex_volume_type == 'io1' and ex_iops:
             params['Iops'] = ex_iops
 
+        if ex_encrypted:
```

Comment:
can you make value checks more explicit please e.g. `if ex_encrypted is not None:`